src/app/components/OptimoPromos/Lists/index.jsx

```diff
@@ -0,0 +1,21 @@
+import { GEL_SPACING_DBL } from '#app/legacy/gel-foundations/src/spacings';
+import styled from '@emotion/styled';
+
+// Bases
+export const StoryPromoLiBase = styled.li`
```

Comment:
In the standards `Give a component a descriptive name rather something too generic or naming a component after an HTML element.` I also don't think `Base` in the naming is necessary.

Reply:
I will rename them. (More on single renaming below)